cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,39 @@
+/* eslint-disable */
 /// <reference types='cypress' />
 
+const { generateUser } = require('../support/generateUser')
 describe('Student Registration page', () => {
   before(() => {
+    cy.visit('https://demoqa.com/automation-practice-form')
+  })
 
-  });
+  it('should send the form with filled fields and check modal', () => {
+    const { userName, email, mobile, subject, currentAddress, state, city } =
+      generateUser()
 
-  it('', () => {
+    cy.findById('firstName').type(userName)
+    cy.findById('lastName').type(userName)
```

Comment:
Cypress does not have a built-in command `findById`. You should use `cy.get('#firstName')` and `cy.get('#lastName')` instead.